src/micromamba/update.cpp

```diff
@@ -37,6 +37,7 @@ set_update_command(CLI::App* subcom)
     init_update_parser(subcom);
 
     subcom->callback([&]() {
+        parse_file_options();
```

Comment:
what happens with the `name` variable of the yaml here?

Reply:
In that case if `-n` is passed to the cli, it will override the one of the yaml file.
But if `-n` is not passed, then it uses the name of the yaml file instead of getting the fallback value from `CONDA_PREFIX` env var.
Sounds good or would it be better to **never** use the env name from yaml file ?

Reply:
usually, `install` and `update` operate on the "active environment". This would somewhat break this assumption (since it would update "another" environment).
That's why I am not sure about what behavior we want.